source/MaterialXGenShader/GenOptions.h

```diff
@@ -93,6 +93,9 @@ class GenOptions
     /// compress a set of UDIMs into a single normalized image for
     /// hardware rendering.
     bool hwNormalizeUdimTexCoords;
+
+    /// Enable texture-space rendering for the given element.
+    bool textureSpaceRender;
```

Comment:
If a new derived shader generator was used for texture baking we would not need an option for this.

Reply:
As it turns out, this flag was the legacy of an earlier implementation, and is no longer needed for texture baking.  :)